### PR TITLE
Add english translations

### PR DESCRIPTION
**Pull request by bertocq**
Connects to #8 

## What does this PR do?
1. Creates folders `es` and `en` for Spanish and English translations of files respectively
2. Moves existing spanish and english versions of files to correct folders: https://github.com/folkswhocode/diversifaq/commit/02f4e20fc41f3a0663acbf6400071e4f4f4b6251  & https://github.com/folkswhocode/diversifaq/commit/f3cd1300dcee96b8e34998a2738ff10737a51f68
3. Adds missing base files from [base project](https://github.com/folkswhocode/base) to each language: https://github.com/folkswhocode/diversifaq/commit/54b7e5a6ffd4eee72b5b5857bf39dcfce273898d & https://github.com/folkswhocode/diversifaq/commit/200edfe3df880276abd171295193450d0248ff23
4. Translates FAQ.md and CONTRIBUTORS.md to English: https://github.com/folkswhocode/diversifaq/commit/2e5b1359740fc8235b3bc0809bcfb811f6aa3a1e & https://github.com/folkswhocode/diversifaq/commit/eefc1c02436c013a802edde103fca5c62da5a3dd as well as the README.md file https://github.com/folkswhocode/diversifaq/commit/c985912aeac757e6fba38315c8861a2eb8e9efef

## Where should the reviewer start?
Commit history should be good guide to follow steps taken

## How should this be manually tested?
Proofreading translations made

## Any background context you want to provide?
None

## What are the relevant issues?
- Organizing english and spanish files in folders following the base project convention
- Adding English and Spanish missing versions of files

## Screenshots (if appropriate)
No need

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [X] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Questions

- [ ] Does the knowledge base (guidelines) need an update?
- [ ] Does this add new dependencies which need to be added to the development environment?
- [ ] Does this add new dependencies which need to be added to the production environment?
- [ ] Is there a blog post?

## By submitting this pull request I confirm I've read and comply with the below requirements

- I have read and understood the [contribution guidelines](https://github.com/folkswhocode/base/blob/master/en/CONTRIBUTING.md)
and our [code of conduct](https://github.com/folkswhocode/base/blob/master/en/CODE_OF_CONDUCT.md).
- This pull request complies with the [code of conduct](https://github.com/folkswhocode/base/blob/master/en/CODE_OF_CONDUCT.md) of the organization.
- This pull request has a descriptive title. For example, `Add new function to calculate user rating`, not `Update readme.md` or `Add some feature`.

### CODE REVIEW

File: en/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# DiversiFAQ: Questions and answers about diversity in technology
+
+### What benefits has a diverse team?
+
+The software we develop inherits our biases. There are many [documentede cases](https://www.theguardian.com/technology/2017/apr/13/ai-programs-exhibit-racist-and-sexist-biases-research-reveals) about this.
```
Comment (dreamingechoes) [inline en/FAQ.md]: I think we have a typo in `documentede cases`, right? 😄

**Review by dreamingechoes — CHANGES_REQUESTED** [review]
This looks awesome! 😍 I just left one comment about a typo, but this looks good to me! But I want @marinavega and @tasugo to have a look at this PR before merge it, just in case 💖😄

File: en/CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Contributors
+
+This is the list of wonderful people that has contributed to this FAQ, listed  es la lista de la gente maravillosa que ha contribuido a este FAQ, ordered alphabetically by their last name:
```
Comment (marinavega) [inline en/CONTRIBUTORS.md]: Languages are mixed here 😅

File: README.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-Preguntas frecuentes sobre diversidad en tecnología.
+Frecuently asked questions about diversity in tech / Preguntas frecuentes sobre diversidad en tecnología.
 
-Consulta el listado de preguntas [aquí](https://github.com/folkswhocode/diversifaq/blob/master/FAQ.md).
+Check the questions list [here](https://github.com/folkswhocode/diversifaq/blob/master/en/FAQ.md). / Consulta el listado de preguntas [aquí](https://github.com/folkswhocode/diversifaq/blob/master/es/FAQ.md).
 
-**En continua actualización.**
+**In continuous update** / **En continua actualización.**
 
-Para añadir tu pregunta y respuesta, haz una pull request a este repositorio añadiéndola al fichero `FAQ.md`, respetando el formato existente. Y no olvides añadirte al fichero de colaboradores, `CONTRIBUTORS.md`.
```
Comment (marinavega) [inline README.md:9]: I find this EN/ES structure a little bit confusing tbh 🤔

**Review by marinavega — CHANGES_REQUESTED** [review]
Thank you so so so much for your time and collaboration ❤️ 
A few changes needed before merging this tho 😸

File: README.md (hunk 3 above)
Comment (bertocq) [reply README.md:9]: Yes I wasn't sure how to tackle the README.md dual language.. as a third or fourth language would make this option really messy. I thought about having a very simple README.md with just links to each language... and a README.md file under each language folder only with that translation. Do you like that option better or what do you have in mind?

File: en/CONTRIBUTORS.md (hunk 2 above)
Comment (bertocq) [reply en/CONTRIBUTORS.md]: Yes my bad 🙈 I should had reviewed more carefully all text after sending the PR. Fixed at https://github.com/folkswhocode/diversifaq/pull/9/commits/68590c2159c9dd003f9730d326afbae43bc865a6 :)

File: en/FAQ.md (hunk 1 above)
Comment (bertocq) [reply en/FAQ.md]: Fixed 🙏 https://github.com/folkswhocode/diversifaq/pull/9/commits/f6784f1f15e1518b6eaeb87dc6ae6edba4171f1a

**Pull request closed without merging**